src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java

```diff
@@ -77,12 +87,14 @@ public VisualEditorPageObject(WebDriver driver) {
   private WebElement mainContent;
   @FindBy(css = ".media-gallery-wrapper.ve-ce-branchNode")
   private WebElement galleryNode;
-  @FindBy(css = ".media-gallery-wrapper.ve-ce-branchNode>div")
-  private List<WebElement> galleryNodes;
   @FindBy(css = ".media-gallery-wrapper.ve-ce-branchNode .toggler")
   private WebElement toggler;
   @FindBy(css = ".ve-ce-surface-highlights-focused .ve-ce-focusableNode-highlight")
   private WebElement focusedHighlight;
+  @FindBy(css = ".ve-init-mw-viewPageTarget-surface")
+  private WebElement veEditorSurface;
```

Comment:
Small typo vEditor would make a bit more sense as "ve" is already visual editor's abbreviation.